server/routes/activitiesMigration/moveActivityStartDateValidator.test.ts

```diff
@@ -0,0 +1,61 @@
+import moment from 'moment'
+import { MoveActivityStartDateForm } from 'express-session'
+import validate from './moveActivityStartDateValidator'
+
+describe('moveActivityStartDateValidator', () => {
```

Comment:
I've only added tests for the new validator. I've done a lot of manual testing running locally but will also add some cypress tests in the next PR.